modules/ROOT/pages/acm-lightning-components.adoc

```diff
@@ -1,270 +1,306 @@
 
 Endpoints are displayed in alphabetical order.
 
 Console components include *API Console*, *API Console Documentation*, *API Console Instances*, *API Console Navigation*, and *API Console Request Panel*. In the Builder, you can add multiple console components to the same page. Changes to a console component are visible in all other console components on that page that display the same API. For example, selecting the *Summary* tab in an API Console results in *Summary* tab selection in all API Consoles on the same page that display the same API.
 
-API:: Choose the API to display.
-Show Documentation Pages:: Show the documentation pages for the API.
-Link Hover Color:: Set the color of links when the cursor hovers over them.
-Button Text Color (Optional):: Set the color of text on buttons.
-Button Hover Text Color (Optional):: Set the color of text on buttons when the cursor hovers over them. Default is the button text color.
-Headline Font Family (Optional):: Set the font family of the API console headline.
-Error Color (Optional):: Set the color of error messages.
-Method Label Get Background Color:: Set the background color of the labels of *GET* methods.
-Method Label Get Color:: Set the color of the labels of *GET* methods.
-Method Label Post Background Color:: Set the background color of the labels of *POST* methods.
-Method Label Post Color:: Set the color of the labels of *POST* methods.
-Method Label Put Background Color:: Set the background color of the labels of *PUT* methods.
-Method Label Put Color:: Set the color of the labels of *PUT* methods.
-Method Label Delete Background Color:: Set the background color of the labels of *DELETE* methods.
-Method Label Delete Color:: Set the color of the labels of *DELETE* methods.
-Method Label Patch Background Color:: Set the background color of the labels of *PATCH* methods.
-Method Label Patch Color:: Set the color of the labels of *PATCH* methods.
-Method Label Options Background Color:: Set the background color of the labels of *OPTIONS* methods.
-Method Label Options Color:: Set the color of the labels of *OPTIONS* methods.
-Method Label Head Background Color:: Set the background color of the labels of *HEAD* methods.
-Method Label Head Color:: Set the color of the labels of *HEAD* methods.
+You can specify values for the following parameters of the *API Console*:
+
+API:: Specifies the API to display.
+Show Documentation Pages:: Shows the API's documentation pages.
+Link Hover Color:: Specifies the hover link color.
+Button Text Color (Optional):: Specifies the color of button text.
+Button Hover Text Color (Optional):: Specifies the color of button hover text. Default is the button text color.
+Headline Font Family (Optional):: Specifies the API console headline font family.
+Error Color (Optional):: Specifies the color of error messages.
+Method Label Get Background Color:: Specifies the *GET* method label background color.
```

Comment:
GLOBALLY unbold GET, POST, PUT, DELETE, and PATCH.

Reply:
GLOBALLY consider whether we really need to document UI elements that are so intuitively named.
Even a very nontechnical person like me can understand the labels without the descriptions.
This is a larger question that applies to a lot of our doc.

Reply:
Unbolded. Agreed, the docs on these properties and the one-item lists need review.